libraries/AP_RangeFinder/AP_RangeFinder_Backend.h

```diff
@@ -48,6 +48,7 @@ class AP_RangeFinder_Backend
     enum Rotation orientation() const { return (Rotation)params.orientation.get(); }
     float distance() const { return state.distance_m; }
     uint16_t distance_cm() const { return state.distance_m*100.0f; }
+    int8_t signal_quality_pct() const { return state.signal_quality_pct; }
```

Comment:
This is public, consider adding `WARN_IF_UNUSED`.  This is similar to C++17's ` [[nodiscard]]` because it ensures that developers consume the return of `signal_quality_pct()`

Reply:
Thanks for the suggestion. I'd prefer not to add the WARN_IF_UNUSED in this case because while ignoring the return value is not useful and probably a coding error, it is not that serious a runtime problem. It would be useful if the compiler could check if the result was consumed and tested for the UNKNOWN value. Failing to test for the UNKNOWN value could cause bugs. But I don't think the compiler will do this sort of a check. I am happy to add WARN_IF_UNUSED if my preference doesn't match other standards or styles.